### PR TITLE
add document for system variable "tidb_enable_fast_table_check"

### PR DESCRIPTION
**Pull request by wjhuang2016**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14185
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1688,6 +1688,17 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - This variable is used to set whether to enable the statistics `Fast Analyze` feature.
 - If the statistics `Fast Analyze` feature is enabled, TiDB randomly samples about 10,000 rows of data as statistics. When the data is distributed unevenly or the data size is small, the statistics accuracy is low. This might lead to a non-optimal execution plan, for example, selecting a wrong index. If the execution time of the regular `Analyze` statement is acceptable, it is recommended to disable the `Fast Analyze` feature.
 
+### `tidb_enable_fast_table_check` <span class="version-mark">Introduced from v7.2.0 version</span>
```
Comment (qiancai) [inline system-variables.md:1691]:
```suggestion
### tidb_enable_fast_table_check <span class="version-mark">New in v7.2.0</span>

```

File: system-variables.md [hunk 2]
```diff
@@ -1688,6 +1688,17 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
+> **Note:**
+>
+> This function does not work for [multi-valued indexes](/sql-statements/sql-statement-create-index.md#multi-valued indexes) and prefixed indexes.
```
Comment (qiancai) [inline system-variables.md:1694]:
```suggestion
> This variable does not work for [multi-valued indexes](/sql-statements/sql-statement-create-index.md#multi-valued indexes) and prefixed indexes.

```

File: system-variables.md [hunk 3]
```diff
@@ -1688,6 +1688,17 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
+- Scope: SESSION | GLOBAL
+- Whether to persist to the cluster: Yes
```
Comment (qiancai) [inline system-variables.md:1696]:
```suggestion
- Persist to the cluster: Yes
```

File: system-variables.md [hunk 4]
```diff
@@ -1688,6 +1688,17 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
+- Type: Boolean
+- Default value: `ON
```
Comment (qiancai) [inline system-variables.md:1698]:
```suggestion
- Default value: `ON`
```

File: system-variables.md [hunk 5]
```diff
@@ -1688,6 +1688,17 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
+- This variable is used to control whether to use a checksum-based approach to quickly check the consistency of data and indexes in a table. The default value `ON` means this feature is enabled by default.
+- When turned on, TiDB executes the [`ADMIN CHECK [TABLE|INDEX]`](/sql-statements/sql-statement-admin-check-table-index.md) statement faster.
```
Comment (qiancai) [inline system-variables.md:1700]:
```suggestion
- When this variable is enabled, TiDB can execute the [`ADMIN CHECK [TABLE|INDEX]`](/sql-statements/sql-statement-admin-check-table-index.md) statement in a faster way.
```

File: system-variables.md (hunk 1 above)
Comment (qiancai) [reply system-variables.md:1691]: To be consistent with descriptions of other variables

File: system-variables.md (hunk 3 above)
Comment (qiancai) [reply system-variables.md:1696]: To be consistent with descriptions of other variables

File: system-variables.md [hunk 6]
```diff
@@ -1688,6 +1688,19 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
 - If the statistics `Fast Analyze` feature is enabled, TiDB randomly samples about 10,000 rows of data as statistics. When the data is distributed unevenly or the data size is small, the statistics accuracy is low. This might lead to a non-optimal execution plan, for example, selecting a wrong index. If the execution time of the regular `Analyze` statement is acceptable, it is recommended to disable the `Fast Analyze` feature.
 
+### tidb_enable_fast_table_check <span class="version-mark">New in v7.2.0</span>
+
+> **Note:**
+>
+> This variable does not work for [multi-valued indexes](/sql-statements/sql-statement-create-index.md#multi-valued indexes) and prefixed indexes.
+
+- Scope: SESSION | GLOBAL
+- Persist to the cluster: Yes
```
Comment (qiancai) [inline system-variables.md:1698]:
```suggestion
- Persists to the cluster: Yes
```

**Review by qiancai — APPROVED** [review]
/lgtm

File: system-variables.md [hunk 7]
```diff
@@ -1688,6 +1688,19 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - This variable is used to set whether to enable the statistics `Fast Analyze` feature.
 - If the statistics `Fast Analyze` feature is enabled, TiDB randomly samples about 10,000 rows of data as statistics. When the data is distributed unevenly or the data size is small, the statistics accuracy is low. This might lead to a non-optimal execution plan, for example, selecting a wrong index. If the execution time of the regular `Analyze` statement is acceptable, it is recommended to disable the `Fast Analyze` feature.
 
+### tidb_enable_fast_table_check <span class="version-mark">New in v7.2.0</span>
+
+> **Note:**
+>
+> This variable does not work for [multi-valued indexes](/sql-statements/sql-statement-create-index.md#multi-valued-index) and prefixed indexes.
```
Comment (Oreoxmt) [inline system-variables.md:1695]:
```suggestion
> This variable does not work for [multi-valued indexes](/sql-statements/sql-statement-create-index.md#multi-valued-index) and prefix indexes.
```

File: system-variables.md [hunk 8]
```diff
@@ -1688,6 +1688,19 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
+
+- Scope: SESSION | GLOBAL
+- Persists to the cluster: Yes
+- Type: Boolean
+- Default value: `ON`
+- This variable is used to control whether to use a checksum-based approach to quickly check the consistency of data and indexes in a table. The default value `ON` means this feature is enabled by default.
```
Comment (Oreoxmt) [inline system-variables.md:1701]:
```suggestion
- This variable is used to control whether to use a checksum-based approach to quickly check the integrity of data and indexes in a table. The default value `ON` means this feature is enabled by default.
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**